### DISCUSSION TITLE
사이드바에 나의링크도 포함시켜줬으면 좋겠습니다.

### DISCUSSION THREAD

**Discussion by bank365** [pronist/hELLO#232] [Ideas]
안녕하세요?
사이드바에 나의링크가 없던데 추가해줬으면 좋겠습니다.
티스토리 기능중의 하나인데 빠져 있는게 먼가아쉽네요 ㅎ
감사합니다

**Comment by pronist** [comment]
@bank365 안녕하세요.

나의 링크 기능은 레거시 기능으로 추가 예정이 없음을 알려드립니다.
